bnd.bnd

```diff
@@ -0,0 +1,3 @@
+Import-Package: !java.lang, !java.util.*, *
+Bundle-Activator: org.apache.sling.commons.logservice.internal.Activator
```

Comment:
It's less error-prone if you use an annotation, as exemplified at https://bnd.bndtools.org/chapters/230-manifest-annotations.html#arbitrary-manifest-headers